rs/bitcoin/ckbtc/minter/src/state/eventlog.rs

```diff
@@ -204,6 +205,23 @@ mod event {
     }
 }
 
+#[derive(Clone, Eq, PartialEq, Debug, Deserialize, Serialize, candid::CandidType)]
+pub struct Event {
+    /// The canister time at which the minter generated this event.
+    pub timestamp: Option<u64>,
+    /// The event type.
+    pub payload: EventType,
+}
+
+impl From<EventType> for Event {
+    fn from(value: EventType) -> Self {
+        Self {
+            timestamp: None,
+            payload: value,
+        }
+    }
+}
```

Comment:
nit: AFAIK this conversion is only needed [L.92](https://github.com/dfinity/ic/blob/437464ba20229205e79efe5ff6b64bc0046ce40e/rs/bitcoin/ckbtc/minter/src/storage.rs#L91) when deserializing legacy events. For that reason I would avoid implementing a trait, which would make this conversion public. but rather inline it L.92 so that new code cannot make use of that conversion.